NSS-Day-Cohort-66/rare-client-extra-medium-1
# Created My Posts

# Comments can now be queried from the database via GET, and a number of expand queries are supported

**New Supported Routes**
1. `/myPosts`

## STEPS TO TEST

1. Pull down the `jafgo1` branch and switch to it
2. Request `http://localhost:8000/myPosts` and verify you see all the current user's posts. Each post should include
- a title
- an author
- a category
3. Posts should be ordered by most recently created at the top
